python-packages/sra_client/sra_client/models/order_schema.py

```diff
@@ -401,7 +401,7 @@ def salt(self, salt):
             raise ValueError(
                 "Invalid value for `salt`, must not be `None`"
             )  # noqa: E501
-        if salt is not None and not re.search(r"^\\d+$", salt):  # noqa: E501
+        if salt is not None and not re.search(r"^\d+$", salt):  # noqa: E501
             raise ValueError(
                 r"Invalid value for `salt`, must be a follow pattern or equal to `/^\\d+$/`"
```

Comment:
Should update these error messages too.  I see there are quite a few of them, so change them all, not just this one.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1731/commits/e066548581514e7a0d7f4667fd081b8239d741aa